docs/supported-technologies.asciidoc

```diff
@@ -52,13 +52,20 @@ To understand the performance characteristics of a web page beyond the page load
 Focusses on the initial rendering and measures the time from when the page starts loading to when any part of the page's content is displayed on the screen. The agent uses the https://www.w3.org/TR/paint-timing/#first-contentful-paint[Paint timing API] available in the browser to capture the timing information. FCP is captured as transaction mark for `page-load` transaction for all chromium-based browsers (Chrome >60, Edge >79, Opera >47, etc.).
 
 `Largest contentful paint (LCP)`::
-A new page-load metric that denotes the time from when the page starts loading to when the critical element (largest text, image, video elements) is displayed on the screen. LCP is available in the browser through 
+A new page-load metric that denotes the time from when the page starts loading to when the critical element (largest text, image, video elements) is displayed on the screen. LCP is available in the browser through
 https://wicg.github.io/largest-contentful-paint/[LargestContentfulPaint API] which relies on the draft https://wicg.github.io/element-timing/[Element Timing API]. LCP is one of the https://web.dev/vitals/[core web vitals] metrics and
 available only in Chrome >77. Captured as transaction mark for `page-load` transaction, maintain LCP within the first *2.5 seconds* of page-load to provide a good user experience.
 
-`First input delay (FID)`::
-FID quantifies the experience of the user when they interact with the page during the page load. It is measured as the time between when a user first interacts with your site (mouse clicks, taps, select dropdowns, etc.) to the time when the
-browser can respond to that interaction. FID is one of the https://web.dev/vitals/[core web vitals] metrics and available only in Chrome >85 via https://wicg.github.io/event-timing/[Event Timing API]. FID is captured as `First Input Delay` span for `page-load` transaction. Maintain FID *below 100 milliseconds* to provide a good user experience.
+Interaction to next paint (INP)::
+INP quantifies responsiveness to user interactions.
+The INP value comes from measuring the latency of all click, tap, and keyboard interactions that happen throughout a single page visit and choosing the longest interaction observed.
+INP is one of the https://web.dev/vitals/[core web vitals] metrics,
+and you can read more about INP in the https://web.dev/articles/inp[web.dev docs].
+To provide a good user experience, Google recommends an INP of *fewer than 200 milliseconds*.
+
+// `First input delay (FID)`::
```

Comment:
FID is still captured by the agent and will still be available on the page metrics. The UI has been updated to showcase the INP values from 8.12.